src/common/workspace-store_test.ts

```diff
@@ -0,0 +1,123 @@
+import mockFs from "mock-fs"
+
+jest.mock("electron", () => {
+  return {
+    app: {
+      getVersion: () => '99.99.99',
+      getPath: () => 'tmp',
+      getLocale: () => 'en'
+    }
+  }
+})
+
+import { WorkspaceStore } from "./workspace-store"
+
+describe("workspace store tests", () => {
+  describe("for an empty config", () => {
+    beforeEach(async () => {
+      WorkspaceStore.resetInstance()
+      mockFs({ tmp: { 'lens-workspace-store.json': "{}" } })
+
```

Comment:
Of course, this is matter of style, but I would remove empty lines here and in other places between consts and `expect` statements.

Reply:
I like to spread out and group like statements. Namely, group all declarations together, all method and function calls together. But put space between groups that are doing separate "actions". So spread out test set up from the test checks and what not. If there are very large blocks of code, I might even put header comments (either a single line, or maybe even a `/** * */` three line block) to specify that what goes above should be treated as a separate mental idea.